besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -2680,6 +2680,46 @@ public void mustVerifyPrivacyIsDisabled() {
     assertThat(enclaveArg.getValue().isEnabled()).isEqualTo(false);
   }
 
+  @Test
+  public void privacyMultiTenancyIsConfiguredWhenConfiguredWithNecessaryOptions() {
+    when(storageService.getByName("rocksdb-privacy"))
+        .thenReturn(Optional.of(rocksDBSPrivacyStorageFactory));
+
+    final URL configFile = this.getClass().getResource("/orion_publickey.pub");
+    parseCommand(
+        "--privacy-enabled",
+        "--rpc-http-authentication-enabled",
+        "--privacy-multi-tenancy-enabled",
+        "--rpc-http-authentication-jwt-public-key-file",
+        "/non/existent/file",
+        "--privacy-public-key-file",
+        configFile.getPath());
+
+    final ArgumentCaptor<PrivacyParameters> privacyParametersArgumentCaptor =
+        ArgumentCaptor.forClass(PrivacyParameters.class);
+
+    verify(mockControllerBuilder).privacyParameters(privacyParametersArgumentCaptor.capture());
+    verify(mockControllerBuilder).build();
+
+    assertThat(privacyParametersArgumentCaptor.getValue().isMultiTenancyEnabled()).isTrue();
+  }
+
+  @Test
+  public void privacyMultiTenancyWithoutAuthenticationFails() {
+    final URL configFile = this.getClass().getResource("/orion_publickey.pub");
```

Comment:
you may avoid it by declaring static final constant variable, but its not a show-stopper.

Reply:
done